### PR TITLE
Issue#101 testresults users/userid/tasks/taskid

### PR DESCRIPTION
**Pull request by ahmadafrasiyab**
This PR is for Issue#101. This code enables the user to get the artillery test results from S3 bucket for the users/{user_id}/tasks/{task_id} endpoint.

### CODE REVIEW

**Comment by ahmadafrasiyab** [comment]
**Pre-requisites**
You will need to have a code editor installed on your local machine with npm.

**Verification**
You will be able to see the table of test results fetched from the S3 bucket after executing the steps below.

**Steps to Run the Script**
1. Pull the code to your local machine.
2. Navigate to /ui/react-ui
3. Run npm install
4. Run npm start
5. You will see the screen below:
image

6. Copy the endpoint listed as **reports/taskid** on the page under **Test Results Supported Endpoints** and paste it in the browser address bar as follows:
```
http://localhost:3000/reports/taskid
```

7. Press Enter, you will see the table of test results in your browser window as follows:

image

**Troubleshooting**
Since the rendering relies on a specific npm module i.e. react-json-to-table, incase of formatting errors please make sure the module is installed correctly.

**Review by farhadbahrehmandhenry — APPROVED** [review]

**Review by andluc21 — APPROVED** [review]
Once the conflicts are resolved , should be good to go!

File: ui/react-ui/src/components/routes/home/Home.js [hunk 1]
```diff
@@ -6,14 +6,23 @@ let Home = () => {
...
+  var TaskIdTestReport = 'reports/taskid';
   return (
     <Container className="text-center mt-5">
       <h2>Supported Endpoints:</h2>
       <hr/>
       <p>{usersEndpoint}</p>
       <p>{userIdEndpoint}</p>
       <p>{userTasksEndpoint}</p>
+
+      <h2>Test Results Supported Endpoints</h2>
```
Comment (farhadbahrehmandhenry) [inline ui/react-ui/src/components/routes/home/Home.js:19]: the only thing that comes to my mind is to add a margin-top to this element, it's too close to the above element.
```<h2 style={{marginTop: '6rem'}}>Test Results Supported Endpoints</h2>```

**Review by farhadbahrehmandhenry — APPROVED** [review]

File: ui/react-ui/src/components/routes/home/Home.js (hunk 1 above)
Comment (ahmadafrasiyab) [reply ui/react-ui/src/components/routes/home/Home.js:19]: Great. Ill resolve this. Thanks

Comment (ahmadafrasiyab) [reply ui/react-ui/src/components/routes/home/Home.js]: added some padding to resolve the issue.

**Comment by ahmadafrasiyab** [comment]
This is now ready for merge. But I would like for the team who worked on react page to verify that I did not break any of their functionality while resolving the conflicts. @David-Prism @farhadbahrehmandhenry @jak-ashuraliev @AllisonBC . If all looks good to you guys, please go ahead and merge. I verified on my end things look ok.

**Review by farhadbahrehmandhenry — APPROVED** [review]
seems, you've resolved all the merge conflicts correctly and successfully. thanks

**Comment by ahmadafrasiyab** [comment]
@farhadbahrehmandhenry  resolved. Please check now.

**Comment by ahmadafrasiyab** [comment]
Merging now. as issues are resolved.

**Pull request merged**